index.html

```diff
@@ -4,9 +4,167 @@
+            </svg>
+            </a>
+            <a href="#">
+            <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M19.7665 0.369765C19.4331 -0.0289051 18.8665 -0.128573 18.4331 0.17043L16.4998 1.49933C15.5998 0.535877 14.3331 0.00431735 12.9665 0.00431735C10.2665 0.00431735 8.03313 2.197 8.03313 4.92125C8.03313 5.31992 8.06646 5.68537 8.16646 6.05081C5.83313 5.55248 4.43313 3.85813 3.8998 0.801657C3.83313 0.402987 3.53313 0.103985 3.13313 0.00431735C2.76646 -0.0953502 2.36646 0.0707624 2.13313 0.369765C2.06646 0.43621 -3.43354 8.07739 4.53313 14.5558C3.76646 16.1837 1.46646 17.579 0.566463 18.0441C0.13313 18.2435 -0.06687 18.7086 0.03313 19.1737C0.13313 19.6388 0.53313 19.9378 0.999797 19.9378H2.73313C12.8665 19.9378 17.4665 14.9212 17.5665 3.69202L19.6665 1.69867C20.0665 1.36644 20.0998 0.768435 19.7665 0.369765Z" fill="#DBD8D7"/>
+            </svg>
+            </a>
+            <a href="#">
+            <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M13.6439 3.03442C10.1551 3.03442 7.28809 5.86201 7.28809 9.37925V19.3103C7.28809 19.6896 7.59897 19.9999 7.97893 19.9999H11.5713C11.9513 19.9999 12.2622 19.6896 12.2622 19.3103V9.65511C12.2622 8.89649 12.8839 8.2758 13.6439 8.2758C14.4038 8.2758 15.0256 8.89649 15.0256 9.65511V19.3103C15.0256 19.6896 15.3364 19.9999 15.7164 19.9999H19.3088C19.6888 19.9999 19.9997 19.6896 19.9997 19.3103V9.37925C19.9997 5.89649 17.1672 3.03442 13.6439 3.03442Z" fill="#DBD8D7"/>
+                <path d="M4.28325 6.27588H0.690846C0.310881 6.27588 0 6.58622 0 6.96553V19.3104C0 19.6897 0.310881 20 0.690846 20H4.28325C4.66321 20 4.97409 19.6897 4.97409 19.3104V6.96553C4.97409 6.58622 4.66321 6.27588 4.28325 6.27588Z" fill="#DBD8D7"/>
+                <path d="M2.48705 0C1.10535 0 0 1.10345 0 2.48276C0 3.86207 1.10535 4.96552 2.48705 4.96552C3.86874 4.96552 4.97409 3.86207 4.97409 2.48276C4.97409 1.10345 3.86874 0 2.48705 0Z" fill="#DBD8D7"/>
+            </svg>
+            </a>
+            <a href="#">
+            <svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#DBD8D7"/>
+            </svg>
+            </a> 
+        </div>
+    </section>
```

Comment:
Kindly use flexbox to make this section responsive and adapt to the screen size. Also please remove the white border around the webpage as shown below:

Your design:
![image](https://user-images.githubusercontent.com/73714615/186863912-2c1706fc-e2a8-4527-88d4-024aa91bf366.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/186863988-70cfe0ff-a3b5-4a14-b588-5c7abe24590a.png)